.github/workflows/awsfulltest.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: seqeralabs/action-tower-launch@v2
+        uses: seqeralabs/action-tower-launch@922e5c8d5ac4e918107ec311d2ebbd65e5982b3d # v2
```

Comment:
```suggestion
        uses: seqeralabs/action-tower-launch@v2
```

Reply:
unfortunately the currently pinned version is broken, so we need to undo this change